src/file.ts

```diff
@@ -64,6 +64,8 @@ import {
 const duplexify: DuplexifyConstructor = require('duplexify');
 import {normalize, objectKeyToLowercase, unicodeJSONStringify} from './util';
 import {GaxiosError, Headers, request as gaxiosRequest} from 'gaxios';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
It would be worth figuring out how to use this library with an `import`.

Did we try:

```
import retry from 'async-retry'
```